mate-panel/panel-run-dialog.c

```diff
@@ -967,9 +1006,18 @@ panel_run_dialog_add_items_idle (PanelRunDialog *dialog)
 	gtk_tree_view_column_set_attributes (column, renderer,
                                              "text", COLUMN_NAME,
                                              NULL);
+	gtk_tree_view_append_column (GTK_TREE_VIEW (dialog->program_list), column);
 
+	renderer = gtk_cell_renderer_accel_new ();
+	g_object_set (renderer, "accel-mode", GTK_CELL_RENDERER_ACCEL_MODE_GTK,
+	             "editable", FALSE, NULL);
+
+	column = gtk_tree_view_column_new_with_attributes ("", renderer,
```

Comment:
You should set a title to the column even if it's not visible, it helps accessibility tools present that properly

Reply:
Added the title "Shortcut".